Source/WebCore/animation/AnimationTimelinesController.cpp

```diff
@@ -52,32 +52,37 @@ DocumentTimelinesController::DocumentTimelinesController(Document& document)
     }
 }
 
-DocumentTimelinesController::~DocumentTimelinesController() = default;
+AnimationTimelinesController::~AnimationTimelinesController() = default;
 
-void DocumentTimelinesController::addTimeline(DocumentTimeline& timeline)
+void AnimationTimelinesController::addTimeline(AnimationTimeline& timeline)
 {
     m_timelines.add(timeline);
 
-    if (m_isSuspended)
-        timeline.suspendAnimations();
-    else
-        timeline.resumeAnimations();
+    if (is<DocumentTimeline>(timeline)) {
```

Comment:
Maybe do an early `return` check here if we're _not_ dealing with a `DocumentTimeline`?

Reply:
Fixed.